### ISSUE TITLE
Switch to SimpleJson

### ISSUE THREAD

**Issue by danielmarbach** [jbogard/NServiceBus.MessageRouting#7]
I guess it would be possible to Switch to simple json and ditch the newtonsoft json dependency

**Comment by jbogard** [comment]
What's Simple Json?

**Comment by alexeyzimarev** [comment]
Newtonsoft.Json is something almost everyone has anyway. Don't you use Newtonsoft.Json in NSB @danielmarbach?

The original Simple.Json is not being updated for three years. We struggle a lot with it in RestSharp and want to get back to Json.Net. Is there any better Simple Json that I don't know about?

**Comment by alexeyzimarev** [comment]
This one? https://github.com/Bunny83/SimpleJSON

**Comment by danielmarbach** [comment]
This one https://github.com/facebook-csharp-sdk/simple-json. We used it in the core but we have moved it out to an external package. Now we are using the source file internalized and shipped as part of the assembly for the InMemoryPersister for example

**Comment by alexeyzimarev** [comment]
Yeah, I think we use that one too in RestSharp. Got like 20 issues related to it. But I have to admit that issues are only valid for advanced scenarios (attributes, etc). But it also can't work properly with lists, for example. https://github.com/restsharp/RestSharp/issues/983

**Comment by danielmarbach** [comment]
The data structure that is serialized is super simple here. The serializer is not even exposed and therefore not controllable in any way, therefore it can almost be considered "internal". That's why I was thinking it could just be replaced.

**Comment by alexeyzimarev** [comment]
Could be, just saying what I know :)

**Comment by danielmarbach** [comment]
Hehe. No problem and really good input
